src/icons/Status.js

```diff
@@ -168,12 +168,14 @@ Loading.propTypes = {
     className: propTypes.string,
 }
 
-export const StatusIconNoDefault = ({
+export const StatusIcon = ({
```

Comment:
@ismay This looks like it contains the changes we discussed as well, except for the "get rid of the nested ternary" comment.

Reply:
Yeah exactly. Maybe we could get rid of the ternary while we're at it? Switching based on the props for example?